site/docs/v1/tech/apis/identity-providers/_link-response-body-base.adoc

```diff
@@ -27,7 +27,7 @@ The token returned from the identity provider. This is treated as an opaque toke
 +
 [NOTE]
 ====
-Prior to version 1.28.0, this value can be retrieved using the link:/docs/v1/tech/apis/registrations.adoc[User Registration APIs]
+Prior to version 1.28.0, this value can be retrieved using the link:/docs/v1/tech/apis/registrations[User Registration APIs]
```

Comment:
You have to add a trailing slash to this, so it should look like this:

> link:/docs/v1/tech/apis/registrations/[User Registration APIs]

Reply:
That is now fixed.